config/settings.yml

```diff
@@ -1657,6 +1657,7 @@ brd:
 
 
 travel_pay:
+  mock: true
```

Comment:
If you check the travel_pay client, I believe we called this `useFakes` or something like that.

Reply:
related to discussion [here](https://github.com/department-of-veterans-affairs/vets-api/pull/16231/files#r1555888844)